k8s/src/test/scala/io/buoyant/k8s/EndpointsNamerTest.scala

```diff
@@ -86,14 +111,13 @@ class EndpointsNamerTest extends FunSuite with Awaits {
         fail(s"unexpected request: $req")
     }
     val api = v1.Api(service)
-    val namer = new EndpointsNamer(Path.read("/test"), api.withNamespace(_))
+    val namer = new EndpointsNamer(Path.read("/test"), api.withNamespace)
 
     @volatile var state: Activity.State[NameTree[Name]] = Activity.Pending
-    val obs = namer.lookup(Path.read("/srv/thrift/sessions")).states respond { s =>
+    val _ = namer.lookup(Path.read("/srv/thrift/sessions")).states respond { s =>
```

Comment:
Question: why do you have to assign this to a val?  It seems like you could just write:

```
namer.lookup(Path.read("/srv/thrift/sessions")).states.respond(state = _)
```

But I think I'm probably misunderstanding what we're trying to do here.  Also, can you make a similar change to this one on line 49 of this file?